connection.go

```diff
@@ -160,6 +166,45 @@ func (sc *snowflakeConn) exec(
 	return data, err
 }
 
+func buildQueryContext(qcc *queryContextCache) (requestQueryContext, error) {
```

Comment:
don't we want to extract these two methods to another file? connection.go is getting rather large.

Reply:
This one is pretty internal. But I agree, that we should reorganise files a bit, but in different commit.